src/components/learner-credit-management/cards/CourseCard.jsx

```diff
@@ -3,95 +3,150 @@
+
+  const enrollmentDeadline = getEnrollmentDeadline(normalized_metadata?.enroll_by_date);
+
+  let courseEnrollmentInfo;
+  if (normalized_metadata?.enroll_by_date) {
+    courseEnrollmentInfo = `${formatAvailability} • ${ENROLLMENT.text} ${enrollmentDeadline}`;
+  } else {
+    courseEnrollmentInfo = formatAvailability;
+  }
+
+  let execEdEnrollmentInfo;
+  if (normalized_metadata?.enroll_by_date) {
+    execEdEnrollmentInfo = `Starts ${formatDate(normalized_metadata.start_date)} •
+    ${ENROLLMENT.text} ${enrollmentDeadline}`;
+  } else {
+    execEdEnrollmentInfo = formatAvailability;
+  }
+
+  const isExecEd = course_type === EXEC_COURSE_TYPE;
+
   return (
     <Card
-      className="course-card"
-      onClick={() => onClick(original)}
-      orientation="horizontal"
-      tabIndex="0"
+      isClickable
+      orientation={isSmall ? 'vertical' : 'horizontal'}
     >
       <Card.ImageCap
-        src={cardImageUrl || cardFallbackImg}
-        fallbackSrc={cardFallbackImg}
-        logoSrc={partners[0]?.logo_image_url}
+        src={imageSrc}
+        logoSrc={logoSrc}
         srcAlt={altText}
         logoAlt={partners[0]?.name}
       />
-      <div className="card-container">
-        <div className="section-1">
-          <p className="mb-1 lead font-weight-bold">{title}</p>
-          <p>{partners[0]?.name}</p>
-          {courseType === EXEC_COURSE_TYPE && (
-            <Badge variant="light" className="mb-4">
-              Executive Education
-            </Badge>
+      <Card.Body>
+        <Card.Header
+          title={title}
+          className="mb-0 mt-0"
+          subtitle={partners[0]?.name}
+          actions={(
+            <Stack gap={1} className="text-right">
+              <p className="h4 mt-2.5 mb-0">{priceText}</p>
+              <span className="micro">{PRICE.subText}</span>
+            </Stack>
           )}
-          {courseType !== EXEC_COURSE_TYPE && (
-            <p className="spacer" />
-          )}
-          <p className={`small ${courseType !== EXEC_COURSE_TYPE ? 'mt-5 mb-0' : ''}`}>
-            Starts {formatDate(normalizedMetadata?.start_date)} •
-            Learner must register by {formatDate(normalizedMetadata?.enroll_by_date)}
-          </p>
-        </div>
-        <Card.Section className="section-2">
-          <p className="lead font-weight-bold mb-0">{priceText}</p>
-          <p className="micro mb-5.5">Per learner price</p>
-          <Card.Footer orientation="horizontal" className="footer">
-            <Button as={Hyperlink} destination="https://edx.org" target="_blank">View course</Button>
-
-            <Button>Assign</Button>
-          </Card.Footer>
+        />
+        <Card.Section>
+          <Badge variant="light" className="ml-0">
+            {isExecEd ? BADGE.execEd : BADGE.course}
+          </Badge>
         </Card.Section>
-      </div>
+        <Card.Footer
+          orientation={isExtraSmall ? 'horizontal' : 'vertical'}
```

Comment:
For the smallest screens, I would think the orientation should be vertical rather than horizontal?

Reply:
We'd want the footer to stack horizontally here per the paragon docs: https://paragon-openedx.netlify.app/components/card/#footer-horizontal-variant

This is what the cards would look like if the orientation is vertical on the smallest screens.
![Screenshot 2023-10-16 at 1 49 06 PM](https://github.com/openedx/frontend-app-admin-portal/assets/71999631/c9c8dee9-ad5b-4870-83ec-2cda7777c11a)